src/js/components/Select/SelectContainer.js

```diff
@@ -36,16 +36,12 @@ const SelectOption = styled(Button)`
   width: 100%;
 `;
 
-const ClearButton = ({ clear, onClear, name, theme, setFocus }) => {
+const ClearButton = ({ clear, onClear, name, theme, innerRef }) => {
```

Comment:
I think this can just use the standard `forwardRef()` from React. That would be a more consistent pattern to follow.